test/mathml-spec.js

```diff
@@ -58,4 +58,10 @@ describe("A MathML builder", function() {
     it('should use <menclose> for colorbox', () => {
         expect(getMathML("\\colorbox{red}{b}")).toMatchSnapshot();
     });
+
+    it('should set href attribute for href appropriately', () => {
+        expect(getMathML("\\href{http://example.org}{\\alpha}")).toMatchSnapshot();
+        expect(getMathML("p \\Vdash \\beta \\href{http://example.org}{+ \\alpha} \\times \\gamma"))
+            .toMatchSnapshot();
+    });
```

Comment:
I didn't realize that you could have two snapshots inside the same `it` block, cool!